src/wp-includes/blocks.php

```diff
@@ -223,8 +225,8 @@ function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
 		if ( ! isset( $metadata['style'] ) ) {
 			$metadata['style'] = "wp-block-$block_name";
 		}
-		if ( ! isset( $metadata['editor_style'] ) ) {
-			$metadata['editor_style'] = "wp-block-$block_name-editor";
+		if ( ! isset( $metadata['editorStyle'] ) ) {
+			$metadata['editorStyle'] = "wp-block-$block_name-editor";
```

Comment:
```suggestion
			$metadata['editorStyle'] = "wp-block-{$block_name}-editor";
```

Wrapping the variable in curly braces helps:

- readability: clearly defines where the variable is within the string it's being concatenated with
- tells PHP where the variable name ends, which is helpful when the variable is injected next to the string characters without spaces (not applicable in this usage)